docs/setup-cluster/checklists/_index.rst

```diff
@@ -315,11 +315,47 @@ Non-Root Containers
 ===================
 
 You can enhance security and limit potential malicious activity by running containers as non-root
-users. To find out more, visit :ref:`run-as-user` and :ref:`run-unprivileged-tasks`.
+users.
 
 .. important::
 
-   Red Hat OpenShift users should not configure non-root containers.
+   Red Hat® OpenShift® users users should not configure non-root containers.
+
+To run containers as non-root users, you'll first need to set up your non-root user:
+
+-  Choose an existing non-root user or create a new one for running containers.
+
+-  Link this user to the actual username/UID and groupname/GID. Use the following command:
+
+   .. code:: bash
+
+      det user link-with-agent-user
+         --agent-user $THE_USER --agent-uid $THE_UID --agent-group $THE_GROUP --agent-gid $THE_GID
+         $THE_DETERMINED_USER
```

Comment:
the missing backslashes at the end of the lines causes this to be three commands instead of just one.

Also as somebody who reads a lot of bash, when there are many options it is common to put each option on its own line, as I suggested before.

Reply:
i re-reviewed it too quickly, i should sloooooow down